README.md

```diff
@@ -68,10 +68,49 @@ To build the lib, run:
 yarn build
 ```
 
+### Generated files
+
+The lib is built on top of some generated files like the design-tokens of the icons fonts. Those generated files are versioned in the `src/generated` and `i18n` folders.
```

Comment:
`like the design-tokens of the icons fonts`: was this `of` meant to be an `or`?

Reply:
I'm not sure it's worth fixing here, but definitely on main.
I made a note of it and will create a PR with these